learnrl/agent.py

```diff
@@ -1,17 +1,18 @@
-
+from learnrl import Agent
```

Comment:
This is mistake, because this will cause circular imports !
Here Agent is defined in that same file !

Reply:
ok got it